examples/live/check_results.py

```diff
@@ -173,7 +173,7 @@ def check_coinc_results(args):
                     and close(inj_mass2[i], trig_props['mass2'][j], 5e-7)
                     and close(inj_spin1z[i], trig_props['spin1z'][j], 5e-7)
                     and close(inj_spin2z[i], trig_props['spin2z'][j], 5e-7)
-                    and close(15.0, trig_props['net_snr'][j], 1.0)):
+                    and close(13.0, trig_props['net_snr'][j], 1.0)):
```

Comment:
@GarethDaviesGW note that there are two injections, and both need to pass this test (one improvement to the test would be to actually calculate the optimal SNRs on its own for each injection, but that is not in place yet).

Reply:
hadnt realised that - Ive updated to use the window of 2 in SNR instead